tools/rosbag/src/rosbag/rosbag_main.py

```diff
@@ -33,6 +33,7 @@
 from __future__ import print_function
 
 import optparse
+import argparse
```

Comment:
optparse has been deprecated in Python 3.2 in favour of argparse. I still tried using optparse in the first place for code consistency here, but optparse lacks nargs='+' support. To the user the difference shouldn't be visible.